packages/server/safeMutations/safeArchiveTeam.ts

```diff
@@ -49,7 +49,7 @@ const safeArchiveTeam = async (teamId: string) => {
     }).run(),
     updateTeamByTeamId(updates, teamId)
```

Comment:
`updateTeamByTeamId` still has a catchAndLog in it (as do a few other files related to the Team table)
could you clean up those remaining files?